samza-core/src/test/java/org/apache/samza/system/inmemory/TestInMemorySystemProducer.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.samza.system.inmemory;
```

Comment:
Can you fix the rat check and add copyright to this new file?

Reply:
Done